unicodetools/org/unicode/tools/emoji/CandidateData.java

```diff
@@ -896,10 +916,10 @@ private static void generateProposalData(CandidateData instance) {
 	//1F931;  L2/16-280,L2/16-282r;   BREAST-FEEDING   
 	Set<String> done = new HashSet<>();
 	UnicodeSet missing = new UnicodeSet();
-	for (String item : instance.allCharacters) {
+	for (String item : instance.fullDraftForProposals) {
```

Comment:
allCharacters was composed by removing characters already in the release; the right thing to do normally, but that means this code wasn't working after beta.